hack/build-client.sh

```diff
@@ -53,8 +53,8 @@ function generate_python() {
     -v $PWD/python/client:/python/client:z \
     -v $PWD/swagger:/swagger:z \
     "${AUTOREST_IMAGE}" \
-    --version=3.4.5 \
-    --use=@autorest/python@5.8.4 \
+    --version=3.5.1 \
+    --use=@autorest/python@5.12.0 \
```

Comment:
where do these versions come from?
should they be variables?
defined somewhere else?

Reply:
These are various Autorest components depending on what SDK you are compiling for. We definitely could define versions as variables, but it's the only place they are used at, so there's not much value. Only makes it harder to run through multiple files to find which versions we run and where to change them in case something is not compiling. That's the reason of bumping up the versions in this PR as it stopped compiling with the original ones we had merged before.

Reply:
Discussed this with Ross and agreed to keep it as is to avoid extra complexity.